.appveyor.yml

```diff
@@ -16,4 +16,11 @@ environment:
     DOTNET_CLI_TELEMETRY_OPTOUT: 1
 test: 'off'
 deploy: 'off'
-os: Visual Studio 2017
+os: Visual Studio 2017 Preview
+before_build:
+  - choco install googlechrome --ignore-checksum
+  - npm install -g selenium-standalone@latest
```

Comment:
We should pin to a specific version to isolate us from breaking changes to "latest".